client/my-sites/backup/wpcom-backup-upsell.tsx

```diff
@@ -1,4 +1,5 @@
-import { Button } from '@automattic/components';
```

Comment:
Some considerations here:
- The `WPCOMUpsellPage` component in this file (`client/my-sites/backup/wpcom-backup-upsell.tsx`) was used only for Jetpack sites and Atomic sites.
- The `WPCOMUpsellPage` component in file `client/my-sites/backup/wpcom-upsell.tsx` was used only for Simple sites.

I noted that they both were quite similar except for some extra content that was appearing in the one for Simple sites. That extra content (highlight the Activity Log feature, explain what is Jetpack) was also useful for Atomic sites, so I decided to centralize both components into one single component (the one inside `client/my-sites/backup/wpcom-backup-upsell.tsx`), ensure that the extra content only appears for Simple and Atomic sites, and use the same component in all sites.